File: backend/springnatour/src/main/java/com/unina/springnatour/dto/route/RouteStopDto.java
package com.unina.springnatour.dto.route;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;
import org.mapstruct.Mapping;

import java.io.Serializable;

@Data
public class RouteStopDto implements Serializable {
    @JsonProperty("stop_id")
    private Long stopId;

    @JsonProperty("stop_number")
    private Integer stopNumber;

    @JsonProperty("stop_latitude")
    private Double stopLatitude;

    @JsonProperty("stop_longitude")
    private Double stopLongitude;
}

File: client/app/src/main/java/com/unina/natourkt/data/remote/dto/RouteStopDto.kt
package com.unina.natourkt.data.remote.dto

import com.google.gson.annotations.SerializedName

data class RouteStopDto(
    @SerializedName("stop_number")
    val stopNumber: Int,

    @SerializedName("stop_latitude")
    val latitude: Double,

    @SerializedName("stop_longitude")
    val longitude: Double,
)
